livingatlas/pipelines/src/main/java/org/apache/beam/sdk/io/solr/SolrIO.java

```diff
@@ -188,27 +187,15 @@ private void populateDisplayData(DisplayData.Builder builder) {
       builder.addIfNotNull(DisplayData.item("username", getUsername()));
     }
 
-    private HttpClient createHttpClient() {
-      // This is bug in Solr, if we don't create a customize HttpClient,
-      // UpdateRequest with commit flag will throw an authentication error.
-      ModifiableSolrParams params = new ModifiableSolrParams();
-      params.set(HttpClientUtil.PROP_BASIC_AUTH_USER, getUsername());
-      params.set(HttpClientUtil.PROP_BASIC_AUTH_PASS, getPassword());
-      return HttpClientUtil.createClient(params);
-    }
-
     AuthorizedSolrClient<CloudSolrClient> createClient() {
-      CloudSolrClient solrClient = new CloudSolrClient.Builder().withZkHost(getZkHost()).build();
+      CloudSolrClient solrClient =
+          new CloudSolrClient.Builder(ImmutableList.of(getZkHost()), Optional.empty()).build();
       solrClient.setZkClientTimeout(180000);
```

Comment:
@djtfmartin - is this the only addition, please?
Let's get this fixed in Beam

Reply:
it is the only addition